yoti_python_sdk/tests/test_anchor.py

```diff
@@ -121,7 +121,10 @@ def test_processing_unknown_anchor_data():
     expected_timestamp = datetime(2019, 3, 5, 10, 45, 11, 840037)
     actual_timestamp = anchors[0].signed_timestamp
 
-    assert expected_timestamp == actual_timestamp
+    try:
+        assert expected_timestamp == actual_timestamp
+    except AssertionError:
+        assert expected_timestamp == actual_timestamp - timedelta(hours=1)
```

Comment:
We need to find a way of doing this independent of timezone

Reply:
yes it will be easlier, let's drop support for python2.7

Reply:
It has been dropped in development, if you do a rebase or a merge you'll get the updated Github actions file which doesn't have 2.7 (looks like there's some conflicts, so I think you'll need to do this anyway)

Reply:
Looks good now :+1: